domain-services/fiscal-event-post-processor/src/main/resources/application.properties

```diff
@@ -39,3 +39,10 @@ ifix.master.project.host=http://localhost:8030
 ifix.master.project.context.path=/ifix-master-data
 ifix.master.project.search.path=/project/v1/_search
 
```

Comment:
Add the topic names from where we are consuming the event and to where we are pushing the event as environment variables. 
We are using following topics: 
1. "fiscal-event-request-validated" 
2. "fiscal-event-request-dereferenced"
3. "fiscal-event-line-item-flattened"